exercises/047_methods.zig

```diff
@@ -2,41 +2,44 @@
 // Help! Evil alien creatures have hidden eggs all over the Earth
 // and they're starting to hatch!
 //
-// Before you jump into battle, you'll need to know four things:
+// Before you jump into battle, you'll need to know three things:
 //
-// 1. You can attach functions to structs:
+// 1. You can attach functions to structs (and other "type definitions"):
```

Comment:
Thank you for the additional clarity here! :+1:

Reply:
I think the official name is *Container*: https://ziglang.org/documentation/master/#Containers